presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java

```diff
@@ -102,34 +102,36 @@ private static TableDefinition partitionedTableWithVariablePartitionsDefinition(
     public void testShowPartitionsFromHiveTable()
     {
         String tableNameInDatabase = tablesState.get(PARTITIONED_TABLE).getNameInDatabase();
+        String partitionsTable = "\"" + tableNameInDatabase + "$partitions\"";
 
         QueryResult partitionListResult;
 
-        partitionListResult = query("SHOW PARTITIONS FROM " + tableNameInDatabase);
```

Comment:
> TestShowPartitions

Consider also renaming the class. (i'm fine if the old name stays though)

Reply:
Thoughts on the new name? `TestPartitionsTable`? It's also the only class in the `convention` package, so I suspect it's in the wrong place.

Reply:
Will rename to `TestHivePartitionsTable` and move to `hive` package.